tasks/jira.py

```diff
@@ -0,0 +1,127 @@
+import os
+import re
+import sys
+
+try:
+    from atlassian import Jira
+except ImportError:
+    pass
+from datadog_api_client import ApiClient, Configuration
+from datadog_api_client.v2.api.ci_visibility_tests_api import CIVisibilityTestsApi
+from datadog_api_client.v2.model.ci_app_aggregate_sort import CIAppAggregateSort
+from datadog_api_client.v2.model.ci_app_aggregation_function import CIAppAggregationFunction
+from datadog_api_client.v2.model.ci_app_compute import CIAppCompute
+from datadog_api_client.v2.model.ci_app_compute_type import CIAppComputeType
+from datadog_api_client.v2.model.ci_app_query_options import CIAppQueryOptions
+from datadog_api_client.v2.model.ci_app_sort_order import CIAppSortOrder
+from datadog_api_client.v2.model.ci_app_tests_aggregate_request import CIAppTestsAggregateRequest
+from datadog_api_client.v2.model.ci_app_tests_group_by import CIAppTestsGroupBy
+from datadog_api_client.v2.model.ci_app_tests_query_filter import CIAppTestsQueryFilter
+from invoke import task
+
+
+def get_failing_tests_names() -> set[str]:
```

Comment:
We should keep only direct `task` methods on the modules in the `tasks` folder. Other ones should take place on the `lib` folder. As a consequence, this file could have a more `test-related` name rather than `jira` to have a `jira` file on `lib`.
I don't have a perfect suggestion, I let you do a proposal